projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts

```diff
@@ -27,6 +27,13 @@ export function checkElement(tabElement: TabElement) {
       cy.focused().should('contain', tabElement.value);
       break;
     }
+    case TabbingOrderTypes.NG_SELECT: {
+      cy.focused()
+        .parentsUntil('ng-select')
+        .last()
+        .parent()
```

Comment:
can we, instead of playing with parents and last elements, go with `.within()` method or something similar?

Reply:
Not really because we need to find a certain parent element where the focused child is contained.